### ISSUE TITLE
[no-passed-in-event-handlers] False-negative with classic components

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/ember-template-lint#2387]
[Reproduction](https://github.com/boris-petrov/ember-4-bug).

Probably no need to go through the complexity of this repo. I think this rule doesn't work with curly-bracked invocations. Doing:

```hbs
  {{#classic-component click=(set this "foobar" true)}}
    click me!
  {{/classic-component}}
```

Doesn't trigger a warning while:

```hbs
<ClassicComponent @click={{set this "foobar" true}}>
  click me!
</ClassicComponent>
```

Does.
